pkg/cmd/server/logs.go

```diff
@@ -93,15 +101,35 @@ func readServerLogFile() error {
 		return fmt.Errorf("failed to get server config: %w", err)
 	}
 
-	file, err := os.Open(cfg.LogFile.Path)
+	logDir := filepath.Dir(cfg.LogFile.Path)
```

Comment:
I believe the user should be able to select files even when not using the `--file` flag. This is important when working with remote servers.

We should have an API that can return a list of log files and the user can then request one to be read over websocket.